app/scripts/controllers/transactions.js

```diff
@@ -26,7 +26,7 @@ angular.module('openhimConsoleApp')
     $scope.settings.list = {};
     $scope.settings.list.tabview = 'same';
     $scope.settings.filter = {};
-    $scope.settings.filter.limit = 10;
+    $scope.settings.filter.limit = 100;
```

Comment:
I think 10 is a good default number as otherwise users may not know see the buttons at the end of the list.